### PR TITLE
convert .format to f-string

### LINKED ISSUE #91: It's time for f-strings
Saw that and decided to wait until after this PR since that goes beyond the scope of the PR.  Let's apply f-strings to the overall code base in a separate issue.

_Originally posted by @prjemian in https://github.com/bluesky/hklpy/pull/85#discussion_r551522630_

### PR DESCRIPTION
**Pull request by prjemian**
fix #91

### CODE REVIEW

**Comment by prjemian** [comment]
@mrakitin : This is ready for review.

File: hkl/engine.py [hunk 1]
```diff
@@ -100,28 +100,28 @@ def limits(self, lims):
...
 
     def _repr_info(self):
-        repr = [
-            "name={!r}".format(self.name),
-            "limits={!r}".format(self.limits),
-            "value={!r}".format(self.value),
-            "fit={!r}".format(self.fit),
-            "inverted={!r}".format(self.inverted),
+        r = [
+            f"name={repr(self.name)}",
```
Comment (mrakitin) [inline hkl/engine.py:104]: Would probably work as `f"name={self.name!r}"`, but either way is fine.

**Review by mrakitin — APPROVED** [review]
Looks good. Have a non-blocking suggestion.

File: hkl/engine.py (hunk 1 above)
Comment (prjemian) [reply hkl/engine.py:104]: will do

Comment (mrakitin) [reply hkl/engine.py]: Thanks, I am happy with the changes.

**Comment by prjemian** [comment]
@mrakitin : Thanks for the review suggestion.  This is more concise.

**Comment by mrakitin** [comment]
I agree, @prjemian, there is no need to apply `repr(...)` everywhere.

**Pull request merged**